client/blocks/product-purchase-features/product-purchase-features-list/jetpack-return-to-dashboard.jsx

```diff
@@ -21,7 +21,7 @@ export default localize( ( { selectedSite, translate } ) => {
 		<div className="product-purchase-features-list__item">
 			<PurchaseDetail
 				icon="house"
-				title={ translate( 'Return to your site\'s dashboard' ) }
+				title={ translate( 'Return to Your Dashboard' ) }
```

Comment:
Not sure if this one is better with title case. Personally I'd use sentence case for it.

Reply:
Fixed!